ehitchhiking-ui/src/app/shared/components/chat/chat.component.sass

```diff
@@ -0,0 +1,136 @@
+@import "../../../../styles/colors"
+@import "../../../../styles/mixins"
+@import "../../../../styles/fonts-constants"
+@import "../../../../styles/constants"
+$image-size: 2rem
+$before-horizontal-radius: 0.5rem
+$before-vertical-radius: 0.4rem
+$message-border-width: 0.625rem
+$after-horizontal-radius: 0.3rem
+$after-width: 0.8rem
+$before-right: -0.2rem
+
+.modal-wrapper
+  .chat
+    overflow-x: hidden
+    .message-zone
+      height: 20rem
+      max-width: 23rem
+      margin: $default-offset/2
+      font-size: $body-2
+      font-weight: normal
+      display: flex
+      flex-direction: column
+.input-zone
+  display: flex
+  justify-content: space-between
+  .input
+    width: 40rem
+  .send-btn
+    background-color: mat-color($primary, 50)
+    height: 2.8rem
+.modal-back
+  position: absolute
+  top: 0.5rem
+  left: 0.5rem
+  @include flexbox()
+  color: mat-color($primary, 400)
+  &-btn:hover
+    background-color: mat-color($accent, 50)
+  &-btn
+    .btn-text
+      display: inline
+
+
+.message-wrapper
+  display: flex
+  justify-content: flex-end
+
+  .message
+    max-width: 8rem
+    word-wrap: break-word
+    margin-bottom: 0.375rem
+    line-height: 1.5rem
+    position: relative
+    padding: 0.6rem 0.7rem
+    border-radius: 1rem
+    background: $light-gray
+    color: $black
+
+    &:before, &:after
+      content: ""
+      position: absolute
+      bottom: -2px
+      height: 0.625rem
+
+  .person-info
+    display: flex
+    flex-direction: row-reverse
+    margin-left: 1.2rem
+    .date
+      font-size: small
+      color: $gray
+      height: 1rem
+      position: fixed
+      margin-top: 2rem
+    .image
+      border-radius: 50%
+      width: $image-size
+      height: $image-size
+
+  &.my-message
+    margin-right: $default-offset/2
+
+    .message
+      color: $white
+      background: $aqua-blue
+      align-self: flex-end
+
+      &:before
+        right: $before-right
+        border-right: $message-border-width solid $aqua-blue
+        border-bottom-left-radius: $before-horizontal-radius $before-vertical-radius
+        transform: translate(0, -2px)
+
+      &:after
+        right: -1.75rem
+        width: $after-width
+        background: $white
+        border-bottom-left-radius: $after-horizontal-radius
+        transform: translate(-1rem, -2px)
```

Comment:
there is the same style for :not(.my-message)

Reply:
done